tests/files/test_ping.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Author - Brennan Paciorek <bpaciore@redhat.com>
+# Description - Benchmark firewalld downtime while reloading and while restarting
+# by measuring how many packets are dropped while firewalld is restarting/reloading
+
+cat > /tmp/Containerfile << EOF
+FROM quay.io/centos/centos:stream8
+RUN dnf -y install systemd
+RUN dnf -y install firewalld nc
+EXPOSE 31337
+CMD /usr/lib/systemd/systemd 
+EOF
+
+trap "rm -f /tmp/Containerfile" EXIT
+
+# Initial container setup #
+{
+  podman network create --subnet 172.16.1.0/24 --gateway 172.16.1.1 --interface-name podmanbr0 podmanbr0
+  trap "podman network rm podmanbr0" EXIT
```

Comment:
hmm - I don't think `trap` is additive - I think this `trap` will erase the previous `trap`:
```
bash -c 'trap "echo trap1" EXIT; trap "echo trap2" EXIT'
trap2
```
If you want to have multiple commands in a `trap`, define a `cleanup` shell function which does all of the cleanup:
```bash
cleanup() {
  rm -f /tmp/Containerfile
  podman network rm podmanbr0 || :
  podman stop test-firewalld || :
  rm -f /tmp/ping0
  rm -f /tmp/ping2
  rm -f /tmp/ping1
}
...
trap "cleanup" EXIT
```

Reply:
You are right, changing this now, just have to debug the cleanup function, since it ignores the `|| :`, most likely due to pipefail